Stocking

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+
+using namespace std;
+
+int main() {
+
+    // Constants
+    int SHARES_BOUGHT=1000;
+    int SHARES_SOLD=1000;
+    double COMMISSION_RATE=.02;
+    double PURCHASE_PRICE=45.5;
+    double SALE_PRICE=56.9;
+
+    // Calculations
+    double purchase_total=SHARES_BOUGHT*PURCHASE_PRICE;
```

Comment:
You should add a space on each side of your operators for readability
that means it should look something like this
```
double sale_total = SHARES_SOLD * SALE_PRICE;
```